index.html

```diff
@@ -75,6 +100,6 @@
 		<path d="M282.8,405.4v-17.5h-7.4h-20.1h-7.4v17.5h4.6c1.6,0,2.8,1.3,2.8,2.8V520c0,1.6-1.3,2.8-2.8,2.8h-4.6v17.5h7.4h20.1h7.4
 		v-17.5h-4.6c-1.6,0-2.8-1.3-2.8-2.8V408.2c0-1.6,1.3-2.8,2.8-2.8H282.8z"/>
 	</svg>
-	<p id="working">Mobile App Developers clearly working</br>on making this web better.</br><a href="mailto:hola@theleftbit.com">hola@theleftbit.com</a></p>
+	<p>Mobile App Developers clearly working</br>on making this web better.</br><a href="mailto:hola@theleftbit.com">hola@theleftbit.com</a></p>
```

Comment:
`working</br>on` is appearing together on the screenshots and it seems like a typo